lib/ts/recipe/emailpassword/components/themes/styles/styles.ts

```diff
@@ -181,6 +181,28 @@ export function getStyles(palette: NormalisedPalette): NormalisedDefaultStyles {
             maxWidth: "96px",
             margin: "0 auto",
         },
+
+        resetPasswordHeaderTitle: {
+            position: "relative",
+            display: "flex",
+            justifyContent: "space-between",
+            alignItems: "center",
+        },
+
+        backButton: {
+            cursor: "pointer",
+            border: "none",
+            backgroundColor: "transparent",
+            width: "15.7px",
```

Comment:
These values are strangely specific, also, I'd prefer if they were just defined in a single place, i.e., a common class for both the placeholder and the actual button.

Reply:
Updated